project/plugins.sbt

```
@@ -6,6 +6,6 @@ resolvers += Resolver.url(
   url("https://artifacts.werally.in/artifactory/ivy-plugins-release")
 )(Resolver.ivyStylePatterns)
 
-addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "latest.release")
+addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "1.3.0")
```

```suggestion
addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "1.10.1")
```
Latest release version https://github.com/AudaxHealthInc/rally-versioning/releases/tag/v1.10.1

---

That was https://github.com/AudaxHealthInc/akka-construct/pull/6/commits/99323a70e4405e922562ae3071f1814e3caed83b which had an issue with the version of git on the build server so I backed up to an older version